application/src/main/java/org/thingsboard/server/service/mail/TbMailSender.java

```diff
@@ -71,24 +70,27 @@ public TbMailSender(TbMailContextComponent ctx, JsonNode jsonConfig) {
         setJavaMailProperties(createJavaMailProperties(jsonConfig));
     }
 
-    @SneakyThrows
     @Override
     public void doSend(MimeMessage[] mimeMessages, @Nullable Object[] originalMessages) {
```

Comment:
Please, do a simple test to ensure that updateOauth2PasswordIfExpired() is called once when doSend()
similar to testConnection() test.